### PR TITLE
Update the README

### PR DESCRIPTION
**Pull request by SergSkol**
Update the README to include a link to the online version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,11 +10,12 @@
 
 ## Live Demo
 
+There is the link to the online version
 https://sergskol.github.io/portfolio
```
Comment (roshan-bajgain) [inline README.md:15]: - [ ] You did a good job but when I try to open with `live-server` then it renders me on some page that says `404`. please fix this issue and submit it again.
### Error
dd

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @SergSkol, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (SergSkol) [reply README.md:15]: Thank you! One letter typo https://sergskol.github.io/Portfolio/  -- the capital P in Portfolio

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark:

Hi @SergSkol, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Good README file. :heavy_check_mark: 
-    App deployed correctly. :heavy_check_mark: 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**